api/src/main/resources/org/openmrs/api/db/hibernate/OrderFrequency.hbm.xml

```diff
@@ -14,38 +14,34 @@
     "-//Hibernate/Hibernate Mapping DTD 3.0//EN"
     "http://www.hibernate.org/dtd/hibernate-mapping-3.0.dtd">
 <hibernate-mapping>
-	<class name="org.openmrs.OrderFrequency" table="order_frequency">
-
-		<id name="orderFrequencyId" type="java.lang.Integer" column="order_frequency_id">
-			<generator class="native">
-				<param name="sequence">order_frequency_id_seq</param>
-			</generator>
-		</id>
-
-		
-
-		<many-to-one name="concept" class="org.openmrs.Concept"
-			not-null="true" column="concept_id" unique="true" />
-		<property name="frequencyPerDay" type="double" column="frequency_per_day"
-			length="22" />
-		<many-to-one name="creator" class="org.openmrs.User"
-			not-null="true" />
-		<property name="dateCreated" type="java.util.Date" column="date_created"
-			not-null="true" length="19" />
-		<property name="retired" type="java.lang.Boolean" length="1"
-			not-null="true" />
-		<many-to-one name="retiredBy" class="org.openmrs.User"
-			not-null="false" column="retired_by" />
-		<property name="dateRetired" type="java.util.Date" column="date_retired"
-			not-null="false" length="19" />
-		<property name="retireReason" type="java.lang.String" column="retire_reason"
-			length="255" />
-		<property name="uuid" type="java.lang.String" column="uuid"
-			length="38" unique="true" />
-
+    <class name="org.openmrs.OrderFrequency" table="order_frequency">
+
+        <id name="orderFrequencyId" type="java.lang.Integer" column="order_frequency_id">
+            <generator class="native">
+                <param name="sequence">order_frequency_id_seq</param>
+            </generator>
+        </id>
+        <many-to-one name="concept" class="org.openmrs.Concept"
+                             not-null="true" column="concept_id" unique="true" />
+        <property name="frequencyPerDay" type="double" column="frequency_per_day"
+                          length="22" />
+        <property name="sortWeight" type="double" column="sort_weight" length="22" />
```

Comment:
My assumption is that hibernate can resolve the field length for primitive types and their wrapper classes